nx-X11/lib/X11/SetLocale.c

```diff
@@ -118,7 +118,7 @@ _Xsetlocale(
 
 #else /* X_LOCALE */
 
-#ifdef __DARWIN__
+#if defined(__APPLE__) || defined(__CYGWIN__)
```

Comment:
DARWIN and APPLE. Good. That makes the code more re-usable. I play with the though of porting it to automake soon.